### PR TITLE
define a default UA so it runs local and travis

### PR DESCRIPTION
**Pull request by iamJoeTaylor**
cc: @square/field-kit-contributors

### CODE REVIEW

File: lib/text_field.js [hunk 1]
```diff
@@ -173,6 +171,7 @@ var TextField = (function (_Input) {
   }, {
     key: '_textDidChange',
     value: function _textDidChange() {
+      debugger;
```
Comment (eventualbuddha) [inline lib/text_field.js]: `debugger`

Comment (eventualbuddha) [inline lib/text_field.js]: I have local changes after running the test:

```diff
diff --git a/lib/text_field.js b/lib/text_field.js
index d2deebc..37b8507 100644
--- a/lib/text_field.js
+++ b/lib/text_field.js
@@ -171,7 +171,6 @@ var TextField = (function (_Input) {
   }, {
     key: '_textDidChange',
     value: function _textDidChange() {
-      debugger;
       var delegate = this._delegate;
       this.textDidChange();
       if (delegate && typeof delegate.textDidChange === 'function') {
```

Comment (iamJoeTaylor) [inline lib/text_field.js]: ewww.. I thought I stopped that, maybe I got the `src` and not `lib`. Nice find.

**Comment by eventualbuddha** [comment]
LGTM

**Pull request merged**